packages/discovery-provider/src/app.py

```diff
@@ -363,7 +363,7 @@ def configure_celery(celery, test_config=None):
             },
             "index_solana_plays": {
                 "task": "index_solana_plays",
-                "schedule": timedelta(seconds=5),
+                "schedule": timedelta(seconds=30),
```

Comment:
is 30 seconds too long for prod? not sure what a good number here would be so we can utilize the cache hits

Reply:
todo: make axiom chart at 5 and see cache hits

Reply:
yeah I think we can leave at 5s and if needed make larger